File: Ejercicios Propuestos 1/Ejercicio 20/Ejercicio 20.py
NAME = "Renzo Leonardo Cienfuegos Cardenas"
GROUP = 2

# Se definen los números triangulares como los obtenidos por la suma de los números naturales sucesivos 1, 2, 3, .... Es decir, los primeros números triangulares son 1, 3, 6, 10, etc. Elabore un programa para imprimir el n-ésimo número triangular.

# Solicitamos el valor de n
n = int(input("n: "))

# Aplicamos la fórmula del número triangular
triangular = n * (n + 1) // 2

# Mostramos el resultado obtenido
print("Número triangular:", triangular)

File: Ejercicios Propuestos 1/Ejercicio 20/Ejercicio20.java
import java.util.Scanner;
public class Ejercicio20 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("¿Cuál es el n-ésimo número triangular? ");
        int n = scanner.nextInt();
        int triangular = 0;
        for (int i = 1; i <= n; i++) {
            triangular += i;
        }
        System.out.println("El número triangular " + n + " es: " + triangular);
    }
}